src/gameboy/CPU.cpp

```diff
@@ -655,6 +740,11 @@ int CPU::execute_instruction() {
             rrc(A);
             F.z = 0; //Special case
             return 1;
+        case 0x10:
+            stop_op();
+            //Says it should take 2 bytes.
+            PC++;
```

Comment:
This is only a typo in the documentation. When running the test rom, `stop` only takes 1 byte